templates/firefox-ios.xliff

```diff
@@ -5262,8 +5322,7 @@ Private Tab</source>
         <source>Load sites lightning fast
 Automatic tracking protection
 Sync on all your devices</source>
-        <note>Subtitle for the Terms of Service screen in the onboarding process. The '
-' symbols denote empty lines separating the first link parameter from the rest of the text.</note>
+        <note>Subtitle for the Terms of Service screen in the onboarding process. The '\n' symbols denote empty lines separating the first link parameter from the rest of the text.</note>
```

Comment:
Made note that `'\n' ` symbol will be parsed in Pontoon, so no need to call out in comments